### DISCUSSION TITLE
labauto elk & filebeat

### DISCUSSION THREAD

**Discussion by haseeb-abdul9** [learndevopsonline/learndevopsonline#1902] [Q&A]
There are a couple of problems with elk and filebeat scripts.
1. Line number 59 in tools/elk/install-v7.sh is faling with error  "No such file or directory".
2. Version 8 script does not open 5044 port.
3. Filebeat script is configured with the repo of version 8 elk.

So no matter what version of the elk script we select it is resulting in failure.

**Comment by chaithanya1812** [comment]
Hi @haseeb-abdul9, join lab session on your free-time I will help you..
